test/interface/dae_initialize_integration.jl

```diff
@@ -35,7 +35,7 @@ p0 = [
 ]
 
 prob = ODEProblem(connected, u0, tspan, p0)
-sol = solve(prob, Rodas5(), initializealg = BrownFullBasicInit())
```

Comment:
why are you changing the initial test?

Reply:
This is an ODE. If you run it with `initializealg` it tells you the kwarg is unsupported.

Reply:
It should be supported. That's a bug that should get fixed then.

Reply:
It's in the supported list: https://github.com/SciML/DiffEqBase.jl/blob/master/src/solve.jl#L61 is this just a spelling error?